packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift

```diff
@@ -392,6 +392,11 @@ public class RNReactNativeGutenbergBridge: RCTEventEmitter {
     func sendEventToHost(_ eventName: String, properties: [AnyHashable: Any]) {
         self.delegate?.gutenbergDidRequestSendEventToHost(eventName, properties: properties)
     }
+
+    @objc
+    func generateHapticFeedback() {
+        UINotificationFeedbackGenerator().notificationOccurred(.success)
```

Comment:
The haptic feedback used in iOS is similar to the one we have for other actions, like for example, [liking a post](https://github.com/wordpress-mobile/WordPress-iOS/blob/a08cc72e4bfb15d4fa88c06c549c16941fb2a64a/WordPress/Classes/ViewRelated/Reader/ReaderLikeAction.swift#L9).

Reply:
Good point, I agree that the current haptic feedback used in iOs might be too intense, I decided to use the `notification` one because I saw several references across the WP-iOS codebase. But, in my opinion, the `selection` one fits way better for this purpose 👍 .

Reply:
Awesome! Let's see if Thomas agrees about changing it or if there are any concerns about it.

Reply:
Is this available in one of the test builds? Happy to take If for a spin!

Reply:
Not yet, but we can push the change and update the installable builds for testing it.

Reply:
Maybe we could change it to the `selection` one and merge this PR with both implementations and if needed then change it back. What do you think?

Reply:
Yeah, I think it would be great to merge it with that option, and eventually in the next phase update it if necessary  👍 . I'll apply the change and request a final review so we could merge it.

Additionally, in the future, we might consider expanding the `generateHapticFeedback` function by providing a configuration argument.

Reply:
Change applied in https://github.com/WordPress/gutenberg/pull/40423/commits/e338a417cbc3a4041f5b9ea2d703d57d59500039.